src/src_V2/utils/constants.js

```diff
@@ -1,4 +1,14 @@
+import { common, grey } from '@mui/material/colors';
+
 export const CONTAINER_SIZES = {
   DRAWER_EXPANDED_WIDTH: 320,
-  DRAWER_COLLAPSED_WIDTH: 33,
+};
+
+export const COLORS = {
+  white: common.white,
+  lightGray: grey[50],
+  midGray: grey[100],
+  darkGray: grey[600],
+  primary: '#2183D1',
+  secondary: 'rgba(0, 0, 0, 0.6)',
 };
```

Comment:
if we decided to move styles on css modules, need we Global Colors @aninaslyan jan? i think we can remove the global COLORS object hym?

Reply:
My point was to use colors with inline style like this BUT move all the other styles related to that element to a css module, because typing color codes in all places is time-consuming and we can have typos.

There is another approach that I think fits the best. We can use [CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) and [CSS import styntax](https://developer.mozilla.org/en-US/docs/Web/CSS/@import) to include colors in each CSS module we need.
colors.css file will look like this:
```
:root {
  --white: #fff;
  --lightGray: #fafafa;
  --midGray: #f5f5f5;
  --darkGray: #757575;
  --primary: #2183d1;
  --secondary: rgba(0, 0, 0, 0.6);
}
```
A CSS module file will have this code at the top of the file:
`@import '../styles/colors.css';`

In this case, we lose the ability to utilize material colors, but still can follow it.
What do you think?

Reply:
I think this is the best one, if this is't cost a lot of time please make changes with this PR. ?